createdisk.sh

```diff
@@ -390,6 +390,12 @@ ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -fr /etc/cni/net.d/200
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo journalctl --rotate'
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo journalctl --vacuum-time=1s'
 
+# Make all Kube-control plane manifest files as immutable. This helps not lose the performance related (ENV and initial resource requests) be overwritten by the respective oeprators
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo chattr +i  /etc/kubernetes/manifests/kube-apiserver-pod.yaml
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo chattr +i  /etc/kubernetes/manifests/kube-controller-manager-pod.yaml
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo chattr +i  /etc/kubernetes/manifests/kube-scheduler-pod.yaml
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- sudo chattr +i  /etc/kubernetes/manifests/etcd-pod.yaml
```

Comment:
Doesn't this start a lot of polling from the operator side since they see the resource are change and every time when try to reconcile it, fails because of `+i` attribute?